flyteidl/protos/flyteidl/service/agent.proto

```diff
@@ -23,7 +23,7 @@ service AsyncAgentService {
   rpc GetTaskMetrics(flyteidl.admin.GetTaskMetricsRequest) returns (flyteidl.admin.GetTaskMetricsResponse){};
 
   // GetTaskLogs returns task execution logs, if available.
-  rpc GetTaskLogs(flyteidl.admin.GetTaskLogsRequest) returns (flyteidl.admin.GetTaskLogsResponse){};
+  rpc GetTaskLogs(flyteidl.admin.GetTaskLogsRequest) returns (stream flyteidl.admin.GetTaskLogsResponse){};
```

Comment:
What does the usage pattern look like here? `GetTaskLogsResponse` contains a list of string results + a page token